packages/graphql/tests/integration/directives/cypher/cypher-with-non-node-types.int.test.ts

```diff
@@ -0,0 +1,316 @@
+                                name
+                            }
+                        }
+                    }
+                }
+            `;
+
+        const gqlResult = await testHelper.executeGraphQL(query);
+
+        expect(gqlResult.errors).toBeUndefined();
+        expect(gqlResult.data).toEqual({
+            [Actor.plural]: [
+                {
+                    name: "Keanu",
+                    movies: [
+                        {
+                            title: "The Matrix",
+                            link: { name: "Main Website", url: "the-matrix.org" },
+                        },
+                    ],
+                },
+            ],
+        });
+    });
+
+    /**
+     * Interfaces without `@node` are not supported yet because there is no way to access "__resolveTree", as the information of the concrete type returned by the cypher field is not available
+     */
+    // eslint-disable-next-line jest/no-disabled-tests
+    test.skip("interface field", async () => {
+        const Movie = testHelper.createUniqueType("Movie");
+        const Link = testHelper.createUniqueType("Link");
+
+        const typeDefs = /* GraphQL */ `
+            type  ${Movie} @node {
+               title: String
+               id: String!
+               link: Link! @cypher(statement: """
+               MATCH(l:${Link})
+               WHERE l.movieId=this.id
+               RETURN l {.name, .url} as link
+               """, columnName: "link")
+            }
+
+            interface Link {
+                movieId: String!
+                url: String!
+                name: String!
+            }
+
+            type ${Link} implements Link {
+                movieId: String!
+                url: String!
+                name: String!
+            }
+        `;
+
+        await testHelper.initNeo4jGraphQL({ typeDefs });
+
+        await testHelper.executeCypher(`
+            CREATE (:${Movie} { id: "matrix", title: "The Matrix"})
+            CREATE (:${Link} {movieId: "matrix", url: "the-matrix.org", name: "Main Website"})
+        `);
+
+        const query = /* GraphQL */ `
+                {
+                    ${Movie.plural} {
+                        title
+                        link {
+                            url
+                            name
+                        }
+                    }
+                }
+            `;
+
+        const gqlResult = await testHelper.executeGraphQL(query);
+
+        expect(gqlResult.errors).toBeUndefined();
+        expect(gqlResult.data).toEqual({
+            [Movie.plural]: [
+                {
+                    title: "The Matrix",
+                    link: { name: "Main Website", url: "the-matrix.org" },
+                },
+            ],
+        });
+    });
+
+    test("query", async () => {
+        const Movie = testHelper.createUniqueType("Movie");
+
+        const typeDefs = /* GraphQL */ `
+            type Movie @node {
```

Comment:
`@node` shouldn't he here right?

```suggestion
            type Movie {
```